assembl/static2/js/app/components/debate/common/post/index.jsx

```diff
@@ -182,12 +182,7 @@ export class DumbPost extends React.PureComponent<Props, State> {
     const translate = contentLocale !== originalLocale;
     const { body, subject, originalBody, originalSubject } = this.getBodyAndSubject(translate);
 
-    const modifiedSubject = (
-      <span>
-        {getSubjectPrefixString(fullLevel)}
-        {subject ? subject.replace('Re: ', '') : ''}
-      </span>
-    );
+    const modifiedSubject = <span>{subject ? subject.replace('Re: ', '') : ''}</span>;
```

Comment:
I think you should inject the span when there is a subject to prevent getting an empty span like that `<span></span>`